functions/memorystore/redis/function.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_memorystore_redis]
+
+package visitcount
```

Comment:
Conventionally, the package name is the same as the directory name. Not a requirement, but maybe we should update either the directory or package name (is there more than one `memorystore` sample where we need that subdirectory?)

Also, please add a package comment (https://github.com/golang/go/wiki/CodeReviewComments#package-comments).

Reply:
The subdirectory is necessary (see https://github.com/GoogleCloudPlatform/golang-samples/tree/master/memorystore/redis for precedent).

I've updated the package name and added a package comment.